lightly_studio/src/lightly_studio/api/routes/api/caption.py

```diff
@@ -38,3 +48,26 @@ def get_caption(
         raise ValueError(f"Caption with ID {caption_id} not found.")
 
     return captions[0]
+
+
+@captions_router.post(
+    "/captions",
+    response_model=CaptionView,
+)
+def create_caption(
+    dataset_id: Annotated[UUID, Path(title="Dataset Id", description="The ID of the dataset")],
```

Comment:
Optional: It would be helpful to remove dataset_id from the parameters, it would save some work later on. Dataset id can be obtained from `parent_sample_id`.

It would need a change to the route.

But I can do that later on.